CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### Fixed
 
+- `Badge`: fixed the unwanted text wrapping ([@driesd](https://github.com/driesd) in [#446](https://github.com/teamleadercrm/ui/pull/446))
```

Comment:
I think description would better be `A badge now no longer wraps text`?

Reply:
Will change!